src/middlewares/requiredFieldsMiddleware.ts

```diff
@@ -103,14 +106,14 @@ function operationRequiresAutosave(operationType: string, actions: Array<Operati
  * @param pendingChanges Pending record changes which could override record values
  * @param rowMeta Fields meta to check for 'required' flag
  */
-function getRequiredFieldsMissing(record: DataItem, pendingChanges: PendingDataItem, fieldsMeta: RowMetaField[]) {
+export function getRequiredFieldsMissing(record: DataItem, pendingChanges: PendingDataItem, fieldsMeta: RowMetaField[]) {
     const result: PendingDataItem = {}
     fieldsMeta.forEach(field => {
         const value = record?.[field.key] as string
         const pendingValue = pendingChanges?.[field.key]
         const effectiveValue = pendingValue !== undefined ? pendingValue : value
         let falsyValue = false
-        if ([undefined, null, ''].includes(effectiveValue as any)) {
+        if ([undefined, null, ''].includes(effectiveValue as string)) {
```

Comment:
`any` conversion is more honest here as conversion to string relies heavily on `strictNullChecks` compiler option

Reply:
I get it